gr-filter/grc/filter_high_pass_filter.block.yml

```diff
@@ -74,6 +75,24 @@ templates:
     - set_taps(firdes.high_pass(${gain}, ${samp_rate}, ${cutoff_freq}, ${width}, ${win},
         ${beta}))
 
+cpp_templates:
+    includes: ['#include <gnuradio/filter/firdes.h>']
+    declarations: 'filter::firdes::sptr ${id};'
+    make: |-
+        this->${id} = filter::${type}::make(
+            ${ interp if str(type).startswith('interp') else decim },
+            firdes.high_pass(
+                ${gain},
+                ${samp_rate},
+                ${cutoff_freq},
+                ${width},
+                ${win},
+                ${beta}));
+    link: ['gnuradio-filter']
+    callbacks:
+    - set_taps(firdes.high_pass(${gain}, ${samp_rate}, ${cutoff_freq}, ${width}, ${win},
+        ${beta}))
+
```

Comment:
We're gonna have to translate `firdes.` to `firdes::` because of the `win` parameter.